components/drops/DropItem.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+  <DropsDropCardSkeleton v-if="loading" />
+
+  <div v-else-if="!loading && pastItem">
+    <DropsDropCard
+      v-if="formattedDrop"
+      :drop="formattedDrop"
+    />
+  </div>
+</template>
+
+<script lang="ts" setup>
+import { getDropAttributes } from './utils'
+import type { DropItem } from '~/params/types'
+
+const props = defineProps<{
+  drop: DropItem
+}>()
+
+const formattedDrop = ref<DropItem>()
+const loading = ref(true)
+const pastItem = computed(() => !formattedDrop.value?.isMintedOut)
+
+onBeforeMount(async () => {
+  formattedDrop.value = await getDropAttributes(props.drop.alias)
```

Comment:
isn't this blocking? can we have it show the skeleton if `!formattedDrop`

Reply:
Should be not blocking, use loading state in there. But, I adjusted it to be able to show minted drops. And it's using your suggestion `!formattedDrop`.